emission/analysis/modelling/tour_model/similarity.py

```diff
@@ -41,10 +41,10 @@ def __init__(self, data, radius, old=True):
                 # print "a is %s" % a
                 t = a
                 try:
-                    start_lon = t.start_loc["coordinates"][0]
-                    start_lat = t.start_loc["coordinates"][1]
-                    end_lon = t.end_loc["coordinates"][0]
-                    end_lat = t.end_loc["coordinates"][1]
+                    start_lon = t.start_place.location["coordinates"][0]

```

Comment:
Don't you need to make a similar change in representatives?

                    points[0].append(c.data.start_loc["coordinates"][1])
                    points[1].append(c.data.start_loc["coordinates"][0])
                    points[2].append(c.data.end_loc["coordinates"][1])
                    points[3].append(c.data.end_loc["coordinates"][0])